src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -444,6 +446,16 @@ public RealJenkinsRule includeTestClasspathPlugins(boolean includeTestClasspathP
         return this;
     }
 
+    /**
+     * Updates the Jenkins-Version defined by the {@code RealJenkinsRuleInit} plugin to match the version of Jenkins that will be run.
+     * The intended use case of this is to prevent any detached plugins being dragged in when they would not normally be.
+     * @param updateRealJenkinsRuleInitPluginBaseline {@code false} to skip updating the plugin, {@code true} (the default) to update the baseline.
+     */
+    public RealJenkinsRule updateRealJenkinsRuleInitPluginBaseline(boolean updateRealJenkinsRuleInitPluginBaseline) {
```

Comment:
Or just do it unconditionally? If there are no known problems, why even offer the option to _not_ do it?

Reply:
https://github.com/jenkinsci/jenkins-test-harness/pull/780#discussion_r1627958793